src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1294,6 +1319,26 @@ else if (v instanceof ListValue)
             if(e instanceof PlayerEntity) ((PlayerEntity) e).getHungerManager().addExhaustion((int) NumericValue.asNumber(v).getLong());
         });
 
+        put("absorption",(e, v)->{
+            if(e instanceof PlayerEntity) ((PlayerEntity) e).setAbsorptionAmount((float) NumericValue.asNumber(v).getLong());
+        });
+
+        put("add_xp",(e, v)->{
+            if(e instanceof PlayerEntity) ((PlayerEntity) e).addExperience((int) NumericValue.asNumber(v).getLong());
+        });
+
+        put("add_xp_level",(e, v)->{
+            if(e instanceof PlayerEntity) ((PlayerEntity) e).addExperienceLevels((int) NumericValue.asNumber(v).getLong());
+        });
+
+        put("score", (e, v)->{
+            if(e instanceof PlayerEntity) ((PlayerEntity) e).setScore((int) NumericValue.asNumber(v).getLong());
+        });
+
+        put("add_score",(e, v)->{
+            if(e instanceof PlayerEntity) ((PlayerEntity) e).addScore((int) NumericValue.asNumber(v).getLong());
+        });
+
```

Comment:
Maybe make style a bit more consistent with the rest (adds spaces) (ignore that in the reviews it seems to have multiple lines):
```suggestion
        put("absorption", (e, v) -> {
            if (e instanceof PlayerEntity) ((PlayerEntity) e).setAbsorptionAmount((float) NumericValue.asNumber(v).getLong());
        });

        put("add_xp", (e, v) -> {
            if (e instanceof PlayerEntity) ((PlayerEntity) e).addExperience((int) NumericValue.asNumber(v).getLong());
        });

        put("add_xp_level", (e, v) -> {
            if (e instanceof PlayerEntity) ((PlayerEntity) e).addExperienceLevels((int) NumericValue.asNumber(v).getLong());
        });

        put("score", (e, v) -> {
            if (e instanceof PlayerEntity) ((PlayerEntity) e).setScore((int) NumericValue.asNumber(v).getLong());
        });

        put("add_score", (e, v) -> {
            if (e instanceof PlayerEntity) ((PlayerEntity) e).addScore((int) NumericValue.asNumber(v).getLong());
        });
```

Reply:
ok, i dont rly care, but whatever.